pkg/blockatlas/client.go

```diff
@@ -73,22 +74,22 @@ func (r *Request) Execute(method string, url string, body io.Reader, result inte
 
 	res, err := r.HttpClient.Do(req)
 	if err != nil {
-		return errors.E(err, errors.TypePlatformRequest, errors.Params{"url": url, "method": method})
+		return errors.E(err, errors.TypePlatformRequest, params)
 	}
 	go metrics.GetMetrics(res.Status, url, method, start)
 
 	err = r.ErrorHandler(res, url)
 	if err != nil {
-		return errors.E(err, errors.TypePlatformError, errors.Params{"url": url, "method": method})
+		return errors.E(err, errors.TypePlatformError, params)
 	}
 	defer res.Body.Close()
 	b, err := ioutil.ReadAll(res.Body)
 	if err != nil {
-		return errors.E(err, errors.TypePlatformUnmarshal, errors.Params{"url": url, "method": method})
+		return errors.E(err, errors.TypePlatformUnmarshal, params)
 	}
 	err = json.Unmarshal(b, result)
 	if err != nil {
-		return errors.E(err, errors.TypePlatformUnmarshal, errors.Params{"url": url, "method": method})
+		return errors.E(err, errors.TypePlatformUnmarshal, params)
```

Comment:
are these errors logged somewhere?

Reply:
in most of the cases in the top-level, inside the handler API response